ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1812,14 +1812,23 @@ class NoteTextComponent extends React.Component {
 				this.props.dispatch({
 					type: 'WINDOW_COMMAND',
 					name: 'commandContentProperties',
-					text: this.rawEditor().session.getValue(),
+					text: this.getSessionText(),
+					lines: this.getSessionLines(),
 				});
 			},
 		});
 
 		return toolbarItems;
 	}
 
+	getSessionText() {
+		return this.rawEditor().session.getValue();
+	}
+
+	getSessionLines() {
+		return this.rawEditor().session.getLength();
+	}
+
```

Comment:
Could you use `this.state.note.body` to get the text. Then `lines = body.split("\n").length`. This is because using `rawEditor` means another hard-dependency to Ace Editor, while we'd like to move away from it, and normally you can have all the required info from this.state.note.